src/components/Heading.astro

```diff
@@ -64,10 +66,34 @@ const { title, subtitle, date, author, authorUrl, authorAvatar, readTime } =
         readTime && (
           <span>
             {" "}
-            <span aria-hidden="true"> ·</span> {readTime}
+            <span aria-hidden="true"> · </span> {readTime}
           </span>
         )
       }
     </p>
+    {
+      editUrl && (
+        <a class="flex dark:text-white pl-1 sm:pl-4 items-center sm:ml-auto font-serif hover:underline" href={editUrl} target="_blank">
```

Comment:
Was there a reason for `font-serif` here as opposed to `font-sans`? This would be the only place on the site where a serif font shows up, so it seems a little out of place to me.

Reply:
Not in particular, I thought that was the other main one. Must have overseen that.